### PR TITLE
[BTCV2-1433] Resolve e2e test todos

### PR DESCRIPTION
**Pull request by dgabka**
## What does this PR do?

Resolve TODO items in e2e test that can be resolved.

## Todos:

- [x] Checked on iOS
- [x] Checked on Android

### CODE REVIEW

**Review by PanRadaCloudbest — APPROVED** [review]
ok

File: tests/e2e/specFiles/Transactions/receive.spec.ts [hunk 1]
```diff
@@ -115,8 +116,8 @@ describe('Transactions', () => {
     describe('Standard P2SH', () => {
       beforeEach(async () => {
         await steps.createWallet({
-          type: 'Standard HD P2SH',
-          name: 'Standard P2SH',
+          type: WalletType.S_HD_P2SH,
```
Comment (aleparty) [inline tests/e2e/specFiles/Transactions/receive.spec.ts:119]: Shouldnt we use WalletType.S_P2SH here? Currently all types are walletType.S_HD_P2SH

Comment (dgabka) [reply tests/e2e/specFiles/Transactions/receive.spec.ts]: Yes, fixed

**Review by aleparty — APPROVED** [review]
LGTM

**Pull request merged**